fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/PushDownProjectLimitRule.java

```diff
@@ -0,0 +1,44 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.sql.optimizer.rule.transformation;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.Lists;
+import com.starrocks.sql.optimizer.OptExpression;
+import com.starrocks.sql.optimizer.OptimizerContext;
+import com.starrocks.sql.optimizer.operator.OperatorType;
+import com.starrocks.sql.optimizer.operator.logical.LogicalLimitOperator;
+import com.starrocks.sql.optimizer.operator.pattern.Pattern;
+import com.starrocks.sql.optimizer.rule.RuleType;
+
+import java.util.List;
+
+/*
+ * e.g.
+ *    Project                    Limit(Global)
+ *        |                           |
+ *   Limit(Global)   =>            Project
+ *        |                           |
+ *       ...                         ...
+ */
+public class PushDownProjectLimitRule extends TransformationRule {
```

Comment:
add comment and todo here.  explain why we need push down project operator to limit operator.